nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/resources/templates/commentReplyNotificationMail.txt

```diff
@@ -0,0 +1,13 @@
+
+(${COMMENT_AUTHOR}) has ${COMMENT_ACTION} a comment on ${COMMENTED_DOCUMENT} at ${COMMENT_DATE}.
+
+says:
```

Comment:
Shouldn't this word be part of a sentence?

Reply:
This is copied after https://github.com/nuxeo/nuxeo/commit/b3f5a8838af3709c9123823ac18aacd526e7d349#diff-d2958c7a23840da67a3967810afb6babR4 which is merged.
My real contribution  is lines [6-9]

Reply:
to give more context about this, i created this template for test, based on the real content of the mail:
```
(Administrator) has added a comment on test at 30/12/2019 - 15:38.


says:

any Comment message


» See all the comments of the document
```